conda_build/metadata.py

```diff
@@ -124,14 +124,17 @@ def get_selectors(config: Config) -> dict[str, bool]:
         linux=plat.startswith("linux-"),
         linux32=bool(plat == "linux-32"),
         linux64=bool(plat == "linux-64"),
+        emscripten=plat.startswith("emscripten-"),
+        wasi=plat.startswith("wasi-"),
         arm=plat.startswith("linux-arm"),
         osx=plat.startswith("osx-"),
-        unix=plat.startswith(("linux-", "osx-")),
+        unix=plat.startswith(("linux-", "osx-", "emscripten-")),
```

Comment:
No `wasi-`?

Reply:
No, afaik wasi is pretty different from Unix. At least that's how it is in the Rust world.